src/Table.tsx

```diff
@@ -185,7 +186,8 @@ export function useDatatableLifecycle({
   }
 
   let data = tableBody;
-  let maxPage;
+  let maxPage = 1;
```

Comment:
I assigned 1 here (I'm not sure it is the right approach), because there is a case right below that maxPage goes undefined.
`if (async === undefined)` and `! if (state.rowsPerPage)`

Reply:
I think it's a right approach, although it's a bit edge case (if it's async then max page should be passed). Still, better safe than sorry. Thanks for noticing that!